bindings/python/iota_sdk/client/_node_core_api.py

```diff
@@ -11,125 +11,161 @@
 
     def get_health(self, url: str):
         """ Get node health.
+
+        Args:
+            url (str): the node's url
         """
         return self._call_method('getHealth', {
             'url': url
         })
 
     def get_node_info(self, url: str, auth=None) -> NodeInfo:
         """Get node info.
+
+        Args:
+            url (str): the node's url
+            auth (optional): JWT or username/password authentication
         """
         return from_dict(NodeInfo, self._call_method('getNodeInfo', {
             'url': url,
             'auth': auth
         }))
 
     def get_info(self) -> NodeInfoWrapper:
-        """Returns the node information together with the url of the used node.
+        """Return node information together with the url of the used node.
         """
         return from_dict(NodeInfoWrapper, self._call_method('getInfo'))
 
     def get_peers(self):
-        """Get peers.
+        """Get the peers of the node.
         """
         return self._call_method('getPeers')
 
     def get_tips(self) -> List[HexStr]:
-        """Get tips.
+        """Request tips from the node.
         """
         return self._call_method('getTips')
 
     def post_block(self, block: Block) -> HexStr:
-        """Post block.
+        """Post a block.
+
+        Args:
+            block (Block): the block to post
+
+        Returns:
+            HexStr: the block id of the posted block
         """
         return self._call_method('postBlock', {
             'block': block.__dict__
         })
 
     def get_block_data(self, block_id: HexStr) -> Block:
-        """Get a block.
+        """Get the block corresponding to the given block id.
         """
         return Block.from_dict(self._call_method('getBlock', {
             'blockId': block_id
         }))
 
     def get_block_metadata(self, block_id: HexStr) -> BlockMetadata:
-        """Get block metadata with block_id.
+        """Get the block metadata corresponding to the given block id.
         """
         return BlockMetadata.from_dict(self._call_method('getBlockMetadata', {
             'blockId': block_id
         }))
 
     def get_block_raw(self, block_id: HexStr) -> List[int]:
-        """Get block raw.
+        """Get the raw bytes of the block corresponding to the given block id.
         """
         return self._call_method('getBlockRaw', {
             'blockId': block_id
         })
 
     def post_block_raw(self, block_bytes: List[int]) -> HexStr:
-        """Post block raw.
+        """Post a block as raw bytes.
+
+        Returns:
+            HexStr: the corresponding block id of the block
         """
         return self._call_method('postBlockRaw', {
             'blockBytes': block_bytes
         })
 
     def get_output(self, output_id: OutputId) -> OutputWithMetadata:
-        """Get output.
+        """Get the output corresponding to the given output id.
+
+        Returns:
+            OutputWithMetadata: the output itself with its metadata
```

Comment:
the rule about typings applies to the `Returns` block too. If you have a type specified, the description should be omitted

Reply:
not the description, but the type you mean, right?